react-ebay-network-aware-code-splitting/src/components/CenterPanelInternal/index.js

```diff
@@ -0,0 +1,67 @@
+
+import React, { useState, lazy, Suspense } from 'react';
+
+import ProductSummary from './ProductSummary';
+import LazyLoadingErrorBoundary from '../LazyLoadingErrorBoundary';
+import AaQAAOSwawpXqRI0_400 from '../../assets/product-images/AaQAAOSwawpXqRI0/s-l400.jpg';
+import AaQAAOSwawpXqRI0_1600 from '../../assets/product-images/AaQAAOSwawpXqRI0/s-l1600.jpg';
+import './center-panel-internal.css';
+
+const Product = lazy(() => {
+  return new Promise(resolve => {
+    navigator.connection ? resolve(navigator.connection.effectiveType) : resolve(null);
+  }).then(
+    effectiveType => {
+      console.log('effectiveType => ', effectiveType);
+      switch (effectiveType) {
+        case '4g':
+          return import(/* webpackChunkName: "magnific-product" */ './MagnificProduct');
```

Comment:
Similar to the feedback earlier, let's update these values: ` return import(/* webpackChunkName: "magnify-product" */ './MagnifyProduct');` etc

Reply:
Sure, I'll update all relevant parts not just sticking to the comments. :)